lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -370,7 +370,9 @@ def elb_healthy(asg_connection, elb_connection, module, group_name):
             if e.error_code == 'InvalidInstance':
                 return None
 
-            module.fail_json(msg=str(e))
+            module.fail_json(msg=str(e), exception=traceback.format_exc() )
+        except socket.gaierror as e:
```

Comment:
I'm not sure about this. 
1) has socket been imported somewhere? 
2) It seems like a random place to include this error. What is the reason for it?